dev.js

```diff
@@ -1,5 +1,6 @@
 require('shelljs/global');
 var fs = require('fs');
+var isWin = ('win32' === require('os').platform());
```

Comment:
> How come you closed this?

Sorry, didn't see your comment earlier. I had to create another pull request for the same change, so closed this one. I see you have accepted the other one. Thanks!